mlflow/server/handlers.py

```diff
@@ -615,6 +615,24 @@ def _create_model_version():
     return _wrap_response(response_message)
 
 
+@catch_mlflow_exception
+def get_model_version_artifact_handler():
+    query_string = request.query_string.decode('utf-8')
+    request_dict = parser.parse(query_string, normalized=True)
+    name = request_dict.get('name')
+    version = request_dict.get('version')
+    artifact_uri = _get_model_registry_store().get_model_version_download_uri(name, version)
+    artifact_repository = get_artifact_repository(artifact_uri)
+    filename = os.path.abspath(artifact_repository.download_artifacts(request_dict['path']))
+    extension = os.path.splitext(filename)[-1].replace(".", "")
+    # Always send artifacts as attachments to prevent the browser from displaying them on our web
+    # server's domain, which might enable XSS.
+    if extension in _TEXT_EXTENSIONS:
+        return send_file(filename, mimetype='text/plain', as_attachment=True)
+    else:
+        return send_file(filename, as_attachment=True)
```

Comment:
I noticed that these lines are repeated verbatim in the handler for `/get-artifact`. Can we DRY this out a bit by introducing an internal `_send_artifact` method?